lib/modules/dosomething/dosomething_rogue/includes/Rogue.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+use DoSomething\Gateway\Common\RestApiClient;
+use DoSomething\Gateway\ForwardsTransactionIds;
+
+class Rogue extends RestApiClient {
+  use ForwardsTransactionIds;
+
+  /**
+   * The class name of the transaction framework bridge.
+   *
+   * @var string
+   */
+  protected $transactionBridge = Phoenix\Auth\PhoenixTransactionBridge::class;
+
+  /**
+   * Create a new Rogue API client.
+   */
+  // public function __construct() {
+  //   $url = ROGUE_API_URL . '/' . ROGUE_API_VERSION . '/';
+
+  //   parent::__construct($url);
+  // }
+
+
+  /**
+   * Send a POST request of the reportback to be saved in Rogue.
+   *
+   * @param string $baseurl
+   * @param array $data
+   * @return object|false
+   */
+  public function postReportback($baseurl, $data) {
+    $response = $this->post(ROGUE_API_URL . '/' . ROGUE_API_VERSION . '/' . $baseurl, $data);
```

Comment:
You shouldn't need to specify `ROGUE_API_URL . '/' . ROGUE_API_VERSION` here because those are set as the base URL when you create the client. I'm also a bit confused by the `$baseurl` parameter here... is there ever a case where we'd want `postReportback` to post to a different path than `reportbacks`?